README.md

```diff
@@ -50,6 +50,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [vite-plugin-tauri](https://github.com/amrbashir/vite-plugin-tauri) - Integrate Tauri in a Vite project to build cross-platform apps.
 - [axios-tauri-adapter](https://git.kaki87.net/KaKi87/axios-tauri-adapter) - `axios` adapter for the `@tauri-apps/api/http` module.
 - [svelte-tauri-filedrop](https://github.com/probablykasper/svelte-tauri-filedrop) - File drop handling component for Svelte.
+- [Astrodon](https://github.com/astrodon/astrodon) - Make desktop apps with Deno 🦕
```

Comment:
```suggestion
- [Astrodon](https://github.com/astrodon/astrodon) - Make tauri desktop apps with Deno 🦕
```
It is not clear how this relates to Tauri so it might be a good idea to throw "tauri" somewhere.

Reply:
Good point! But, just to be clear, while at the moment we are using tauri via ffi we will move to using `wry` (see https://github.com/marc2332/deno_tauri). So, we will not be using tauri directly. I am okay with saying `Make tauri desktop apps with Deno 🦕` anyway :-). 

Is that okay for you?

Reply:
Capitalize Tauri and we're hood IMO 👍🏻

Reply:
> Good point! But, just to be clear, while at the moment we are using tauri via ffi we will move to using `wry` (see https://github.com/marc2332/deno_tauri). So, we will not be using tauri itself. I am okay with saying `Make tauri desktop apps with Deno 🦕` btw :-). 
> 
> Is that okay for you? 

Once `Astroden` migrates to `wry` we can change the description to "Build desktop apps with deno backed by wry" or something.